prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/TN/TN-receiver-transform.yml

```diff
@@ -0,0 +1,49 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  MSH: '/MSH'
+
+elements:
+  - name: tn-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: tn-sending-application-namespace-id
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ '%{MSH}-3-1']
+
+  - name: tn-sending-application-universal-id
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ '%{MSH}-3-2' ]
+
+  - name: tn-sending-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-3-3' ]
+
+  - name: tn-receiving-application-namespace-id
+    value: [ '"tdh"' ]
```

Comment:
This value is "tdh-ELR"

Reply:
That was not what they had in their settings.

Reply:
@JessicaWNava  It looks like they have tdh-ELR for MSH-5 and TDH for the batch headers FHS-5 and BHS-5.

In the covid pipeline and in the UP. These settings are only to set the values in the batch headers:
```
  "receivingApplicationName": "tdh-ELR",
  "receivingApplicationOID": "2.16.840.1.113883.3.773.1.1.3",
  "receivingFacilityName": "TDH",
  "receivingFacilityOID": "2.16.840.1.113883.3.773",
```
For MSH-5 you need to look at the replaceValueAWithB setting:
```
"replaceValueAwithB": {
    "MSH-3": [
      {
        "*": "CDC PRIME - Atlanta, Georgia (Dekalb)^2.16.840.1.114222.4.1.237821^ISO"
      }
    ],
    "MSH-5": [
      {
        "*": "tdh-ELR^2.16.840.1.113883.3.773.1.1.3^ISO"
      }
    ],
    "MSH-6": [
      {
        "*": "TDH^2.16.840.1.113883.3.773^ISO"
      }
    ]
```